src/config/Config.ts

```diff
@@ -167,7 +170,7 @@ const DEFAULT_PROPERTIES: Properties = {
     'hazelcast.invalidation.reconciliation.interval.seconds': 60,
     'hazelcast.invalidation.max.tolerated.miss.count': 10,
     'hazelcast.invalidation.min.reconciliation.interval.seconds': 30,
-    'hazelcast.logging.level': 'INFO',
+    'hazelcast.logging.level': LogLevel.INFO,
```

Comment:
Sorry to be paranoid but I have to come back to this again :smile: 

So, if I have a code like this, that won't work, right?

```
const c1 = await newClient({properties:{"hazelcast.logging.level": "INFO"}});
const c2 = await newClient(c1.getConfig());
```

I think to make this work, we either
- Revert the changes we did (LoggingService etc.)
- Make it accept values of the enum as well

Reply:
I think reverting it is the way to go. And tryGetEnum will be used to just verify it. I think it is ok the code to be asymmetric there :)

Reply:
I did not know we have such an API(getConfig), so thanks for the heads up